corehq/apps/cloudcare/static/cloudcare/js/form_entry/entries.js

```diff
@@ -388,21 +395,9 @@ hqDefine("cloudcare/js/form_entry/entries", function () {
         self.templateType = 'select';
         self.choices = question.choices;
         self.isMulti = true;
-        self.hideLabel = ko.observable(options.hideLabel);
-
-        // A custom binding to add a column layout class only if hideLabel = true
-        ko.bindingHandlers.addColStyle = {
-            update: function(element, valueAccessor) {
-                var value = valueAccessor();
-                var shouldAddColStyle = ko.unwrap(value);
-
-                if (shouldAddColStyle) {
-                    // Account for number of choices plus column for clear button
-                    var colWidth = parseInt(12 / (self.choices().length + 1)) || 1;
-                    $(element).addClass('col-xs-' + colWidth)
-                }
-            }
-        }
+        self.hideLabel = options.hideLabel;
+
+        self.colStyleIfHideLabel = self.hideLabel ? self.getColStyle(self.choices().length) : null;
```

Comment:
I think `colStyleIfHideLabel` should be a computed observable, because the length of `choices` can change - you can have a multiple choice question that's based on a lookup table and then have a filter on the table that involves another question in the form.

Reply:
Wow, great observation. Change made, you'll notice I made the `colStyle` in `ChoiceLabelEntry` an observable as well.